tasks/omero-install.yml

```diff
@@ -36,7 +36,7 @@
     {{ omero_server_omego_options }}
     {{ omero_server_omego_db_options }}
     --rootpass {{ omero_server_rootpassword | quote }}
-    --initdb
+    {{ omero_server_database_manage | ternary('--initdb', '') }}
```

Comment:
One semantics question: if `--initdb` is not passed, does `--rootpass ROOTPASS` effectively becomes a no-op?